docs/antctl.md

```diff
@@ -66,6 +66,12 @@ antctl get controllerinfo
 antctl get agentinfo
 ```
 
+On Windows Nodes, an environment variable `$env:POD_NAME` needs to be set when printing the runtime information of
```

Comment:
Is it just for antctl to detect it should run in agent mode? Maybe we should add a cmd line argument for agent or controller mode, and even specify which Node to run agent commands since now agent API supports remote access too. @weiqiangt, @lzhecheng : what you think?

And do you think possible to run antctl in the management Pod?

Reply:
The environment variable is to help antctl identify it is running in agent mode.

If we run antctl in the management Pod on Windows, it will be more complicated. For the Antrea Agent process is actually running on the host(the listening Port is on the host), if we run antctl in the Pod, we might need another named pipe to forward the commands to host again to connect to the Agent.

Reply:
> The environment variable is to help antctl identify it is running in agent mode.
> 
> If we run antctl in the management Pod on Windows, it will be more complicated. For the Antrea Agent process is actually running on the host(the listening Port is on the host), if we run antctl in the Pod, we might need another named pipe to forward the commands to host again to connect to the Agent.

Can we add a flag to tell antctl it is running in agent mode?

Reply:
Yes, let us add a flag. Maybe antctl can by default connect to the current Node's IP (or loopback IP on Linux) when the agent mode flag is set.